### PR TITLE
Add wrappers for vg and wgsim

### PR DESCRIPTION
**Pull request by cartoonist**
I created the same PR a while ago when the repository was on BitBucket. It was stalled because vg was not on conda at that time. Now, they are on conda. I have updated those wrappers with the latest version of vg and wgsim.

### CODE REVIEW

File: bio/vg/construct/environment.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+channels:
+  - bioconda
+  - defaults
+dependencies:
+  - vg >=1.27.0
```
Comment (johanneskoester) [inline bio/vg/construct/environment.yaml:5]:
```suggestion
  - vg ==1.27.0
```
We never know what happens in the future, hence wrappers are usually pinned to an exact version.

**Review by johanneskoester — CHANGES_REQUESTED** [review]
Awesome, I am really happy to finally being able to merge this. Just one request about the version in each of the conda envs below.

File: bio/vg/construct/environment.yaml (hunk 1 above)
Comment (cartoonist) [reply bio/vg/construct/environment.yaml]: That makes sense. I pinned `vg` and `wgsim` to the exact version.

**Comment by cartoonist** [comment]
Just a kind reminder. I applied the requested changes.

**Review by johanneskoester — APPROVED** [review]

**Comment by johanneskoester** [comment]
Thanks a lot for the reminder!

**Pull request merged**